src/main/java/com/nonononoki/alovoa/service/ScheduleService.java

```diff
@@ -63,9 +66,21 @@ public void scheduleLong() {
 		if (enableSchedules) {
 			Date date = new Date();
 			cleanContact(date);
+			cleanNonConfirmedUsers(date);
 		}
 	}
 
+	public void cleanNonConfirmedUsers(final Date date) {
+		long ms = date.getTime();
+		ms -= nonConfirmedUsersCleanupDelay;
+		Date d = new Date(ms);
+
+		List<User> users = userRepo.findByConfirmedIsFalseAndDatesCreationDateBefore(d);
+
+		userRepo.deleteAll(users);
```

Comment:
Could be a good idea to double check before purging user... you know, to avoid risking purging the whole database in the case of "userRepo.findByConfirmedIsFalseAndDatesCreationDateBefore" not being restrictive enough. Btw, were is this query implemented? I can't find it on the PR diff.